src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -52,6 +52,10 @@ public static function castInterval(\DateInterval $interval, array $a, Stub $stu
 
     private static function formatInterval(\DateInterval $i)
     {
+        if ($i->m >= 12 || $i->d >= 31 || $i->h >= 24 || $i->i >= 60 || $i->s >= 60) {
```

Comment:
are you sure about `$i->m >= 12` ? "One month" is different than "30 days" (or 28,29,31)